visualize-admin/visualization-tool
# fix: Dataset preview when every observation contains undefined values

Fixes #1341.

This PR fixes dataset preview in cases when every observation contains undefined value.

Initially, the idea was to show "nice" previews by filtering out observations with undefined values; however it might happen that every observation will have at least one undefined value – we have to remove this condition to allow displaying such cubes.

### How to test
1. Go to this link (PR). (TODO once deployment finishes)
2. See that the preview is shown.
3. Go to [this link](https://test.visualize.admin.ch/browse?previous=%7B%22includeDrafts%22%3Atrue%7D&dataset=https%3A%2F%2Fenvironment.ld.admin.ch%2Ffoen%2Fubd01041prod%2F3&dataSource=Prod) (TEST).
4. See that the preview is broken.
